curriculum/challenges/english/08-coding-interview-prep/rosetta-code/ludic-numbers.md

```diff
@@ -0,0 +1,98 @@
+---
+id: 5ea281203167d2b0bdefca00
+title: Ludic numbers
+challengeType: 5
+---
+
+## Description
+
+<section id='description'>
+<a href="https://oeis.org/wiki/Ludic_numbers" about="_blank">Ludic numbers</a> are related to prime numbers as they are generated by a sieve quite like the <a href="http://rosettacode.org/wiki/Sieve of Eratosthenes" about="_blank">Sieve of Eratosthenes</a> is used to generate prime numbers.
```

Comment:
Let's link to the secure versions of each page, and use `target="_blank"`:

`<a href="https://oeis.org/wiki/Ludic_numbers" target="_blank">Ludic numbers</a> are related to prime numbers as they are generated by a sieve quite like the <a href="https://rosettacode.org/wiki/Sieve_of_Eratosthenes" target="_blank">Sieve of Eratosthenes</a> is used to generate prime numbers.`